Gems/RemoteTools/Code/Source/RemoteToolsEditorSystemComponent.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <RemoteToolsEditorSystemComponent.h>
+
+namespace RemoteTools
+{
+    void RemoteToolsEditorSystemComponent::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<RemoteToolsEditorSystemComponent, RemoteToolsSystemComponent>()
+                ->Version(0);
+        }
+    }
+
+    RemoteToolsEditorSystemComponent::RemoteToolsEditorSystemComponent() = default;
+
+    RemoteToolsEditorSystemComponent::~RemoteToolsEditorSystemComponent() = default;
+
+    void RemoteToolsEditorSystemComponent::GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided)
+    {
+        BaseSystemComponent::GetProvidedServices(provided);
+        provided.push_back(AZ_CRC_CE("RemoteToolsEditorService"));
+    }
+
+    void RemoteToolsEditorSystemComponent::GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible)
+    {
+        BaseSystemComponent::GetIncompatibleServices(incompatible);
+        incompatible.push_back(AZ_CRC_CE("RemoteToolsEditorService"));
+    }
+
+    void RemoteToolsEditorSystemComponent::GetRequiredServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& required)
+    {
+        BaseSystemComponent::GetRequiredServices(required);
+    }
+
+    void RemoteToolsEditorSystemComponent::GetDependentServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& dependent)
+    {
+        BaseSystemComponent::GetDependentServices(dependent);
+    }
+
+    void RemoteToolsEditorSystemComponent::Activate()
+    {
+        RemoteToolsSystemComponent::Activate();
```

Comment:
BaseSystemComponent::

Reply:
I think this file may have been generated during Gem creation and I never noticed it. It's actually not in the cmake file lists. Deleting it and its header.